syntax = "proto3";

package connection;

service ConnectionService {
    rpc CreateAccount(CreateAccountRequest) returns (CreateAccountResponse);
    rpc Login(LoginRequest) returns (LoginResponse);
    rpc ClientDisconnected(ClientDisconnectedRequest) returns (ClientDisconnectedResponse);
}

message CreateAccountRequest {
    string username = 1;
    string password = 2;
}

message CreateAccountResponse {
    string message = 1;
}

message LoginRequest {
    string username = 1;
    string password = 2;
}

message LoginResponse {
    string message = 1;
}

message ClientDisconnectedRequest {
}

message ClientDisconnectedResponse {
}